src/models/toil-accrual.model.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiToilAccrual } from '../interfaces/toil-accrual.interface';
+
+export class ToilAccrual {
+  public comments: string;
+  public created_at: number;
+  public created_by: number;
+  public date: string;
+  public deleted: boolean;
+  public deleted_at?: string;
+  public deleted_by?: string;
+  public duration_hours: number;
+  public id: number;
+  public leave_year: number;
+  public location_id: number;
+  public user_id: number;
```

Comment:
Super minor as I think we will drop these model classes at some point in the future but is the `public` keyword used on other model classes? I'd be in favour of dropping these but if they are present on other model classes, I'm happy to leave them

Reply:
They do all have them